site/content/docs/main/contributions/tencent-config.md

```diff
@@ -0,0 +1,184 @@
+
+There are other installation parameters that can be viewed using `velero install --help`, such as setting `--use-volume-snapshots-false`  to close the storage volume data snapshot backup if you do not want to back up the storage volume data.
+
+After executing the installation commands above, the installation process looks like this:
+
+{{< figure src="https://main.qcloudimg.com/raw/9015313121ed7987558c88081b052574.png" width="100%">}}
+
+After the installation command is complete, wait for the velero and restic workloads to be ready to see if the configured storage location is available.
+
+Executing the 'velero backup-location get' command to view the storage location status and display "Available" indicates that access to Tencent Cloud COS is OK, as shown in the following image:
+
+{{< figure src="https://main.qcloudimg.com/raw/69194157ccd5e377d1e7d914fd8c0336.png" width="100%">}}
+
+At this point, The installation using Tencent Cloud COS as Velero storage location is complete, If you need more installation information about Velero, You can see the official website [Velero documentation](https://velero.io/docs/) .
+
+## Velero backup and restore example
+
+In the cluster, use the helm tool to create a minio test service with a persistent volume, and the minio installation method can be found in the [minio installation](https://github.com/minio/charts), in which case I have bound a load balancer for the minio service to access the management page using a public address in the browser.
+
+{{< figure src="https://main.qcloudimg.com/raw/f0fff5228527edc72d6e71a50d5dc966.png" width="100%">}}
+
+Sign in to the minio web management page and upload some image data for the test, as shown below:
+
+
+{{< figure src="https://main.qcloudimg.com/raw/e932223585c0b19891cc085ad7f438e1.png" width="100%">}}
+
+With Velero Backup, you can back up all objects in the cluster directly, or filter objects by type, namespace, and/or label, and in this example I use the following command to back up only all resources under the 'default' namespace:
+
+```
+velero backup create default-backup --include-namespaces <Namespace>
+```
+
+Use the `velero backup get` command to see if the backup task is complete, and when the backup task status is "Completed," the backup task is completed without any errors, as shown in the following below:
+
+{{< figure src="https://main.qcloudimg.com/raw/eb2bbabae48b188748f5278bedf177f1.png" width="100%">}}
+
+
+At this point we delete all of minio's resources, including its PVC persistence volume, as shown below:
+
+{{< figure src="https://main.qcloudimg.com/raw/15ccaacf00640a04ae29ceed4c86195b.png" width="100%">}}
+
+After deleting the minio resource, we can test using the previous backup to restore the deleted minio resource, and temporarily update the backup storage location to read-only mode (this prevents the backup object from being created or deleted in the backup storage location during the restore process):
```

Comment:
suggestion - "After deleting the MinIO resource, use your backup to restore the deleted MinIO resource, and temporarily update the backup storage location to read-only mode (this prevents the backup object from being created or deleted in the backup storage location during the restore process):"

Reply:
thank u, I will fixed it.